.github/actions/setup-tebako/action.yml

```diff
@@ -159,8 +159,11 @@ runs:
     - name: Install brew packages
       if: runner.os == 'macOS'
       shell: bash
-      run: | 
-        brew bundle
+      run: |
```

Comment:
This was a bug
Please note the "latest" in this repo  is not tagged version, so it does not correspond to any published gem version.

Reply:
But it's ok to use latest/main `Brewfile` right?